src/Altinn.App.Api/Controllers/ProcessController.cs

```diff
@@ -299,6 +330,41 @@ private async Task<bool> CanTaskBeEnded(Instance instance, string currentTaskId,
             }
         }
 
+        private ActionResult<AppProcessState> GetResultForError(ProcessChangeResult result)
```

Comment:
Wouldn't it be more correct with:
```suggestion
        private ActionResult<ProblemDetails> GetResultForError(ProcessChangeResult result)
```

Reply:
Dette funka ikke, reverter.